ItalianTranslation_PlanetCoaster.xml

```diff
@@ -5277,7 +5277,7 @@
 		<entry key="BuildingPartName_SH_Food_Mexican" translation="Mexelente"/>
 		<entry key="BuildingPartName_SH_Food_Pizza" translation="Pizza Pen"/>
 		<entry key="BuildingPartName_SH_Food_Premium" translation="Chiosco Cibo Premium"/>
-		<entry key="BuildingPartName_SH_Hats" translation="Negozio di Cappelli"/>
+		<entry key="BuildingPartName_SH_Hats" translation="Hat's Fantastic"/>
```

Comment:
Qui torna in inglese what

Reply:
Si, ho preferito lasciare in inglese i nomi dei negozi, così come le attrazioni. Penso sia sensato non tradurli alla lettera.. Oppure no?

Reply:
No credo sia meglio tradurli personalmente

Reply:
QUindi qualcosa come "Cappelli fantastici"

Reply:
Cappelli fantastici mi sembra il più adatto